src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -158,6 +162,43 @@ private function doProcessValue(mixed $value, bool $isRoot = false): mixed
         return $value;
     }
 
+    private function processAttributes(TypedReference $reference): mixed
+    {
+        if (!$attributes = $reference->getAttributes()) {
+            return $reference;
+        }
+
+        foreach ($attributes as $attribute) {
```

Comment:
Currently only one attribute is allowed. But it made me think, is there a use-case for multiple attributes?

Reply:
With the set of attributes we have now, I can't see why/how you'd use multiple.

Should I change `SubscribedService` to only accept a single attribute?

Reply:
Let's keep it future-proof.